src/winetricks

```diff
@@ -7310,12 +7325,11 @@ w_metadata dxvk dlls \
 load_dxvk()
 {
     # https://github.com/doitsujin/dxvk
-    # There's no stable exe URL, but they do provide a RELEASE file that lets us build one:
-    w_download_to "${W_TMP_EARLY}" "https://raw.githubusercontent.com/doitsujin/dxvk/latest-release/RELEASE"
-    dxvk_version="$(cat "${W_TMP_EARLY}/RELEASE")"
-    w_linkcheck_ignore=1 w_download "https://github.com/doitsujin/dxvk/releases/download/v${dxvk_version}/dxvk-${dxvk_version}.tar.gz"
-    helper_dxvk "dxvk-${dxvk_version}.tar.gz" "d3d10_enabled" "3.19" "1.1.88"
-    unset dxvk_version
+	w_download_to "${W_TMP_EARLY}" "https://api.github.com/repos/doitsujin/dxvk/releases/latest" "" "release.json"
+    _W_dxvk_version="$(awk -F':' '{ gsub("[\" ,]",""); if ($1 == "tag_name") {sub("^v","",$2); print $2; exit} }' "${W_TMP_EARLY}/release.json")"
```

Comment:
Not necessarily a blocker, but this is broken on aria2c, see https://github.com/aria2/aria2/issues/1389.

I was thinking we could make a helper for github API release checks, then have two codepaths, one for curl/wget, one for aria2c (not sure about fetch).